dbt/adapters/default/impl.py

```diff
@@ -216,6 +246,13 @@ def rename(self, schema, from_name, to_name, model_name=None):
 
     def rename_relation(self, from_relation, to_relation,
                         model_name=None):
+        if dbt.flags.USE_CACHE:
+            self.cache.rename(
```

Comment:
same as above, I think this should just operate on relations, not their schemas/identifiers